addons/calendar/static/src/views/attendee_calendar/attendee_calendar_model.js

```diff
@@ -192,5 +192,18 @@ export class AttendeeCalendarModel extends CalendarModel {
         }
         await this.load();
     }
+
+    async createFilter(fieldName, filterValue) {
+        const info = this.meta.filtersInfo[fieldName];
+        if (info && info.writeFieldName && info.writeResModel) {
+            const dataArray = filterValue.map(value => ({
```

Comment:
During my test, filterValue is an integer, map can't be applied to it.

Reply:
I fixed it. The issue occurred because `createFilter()` is called from `createSelectDialog`, which expects it to return an array of multiple IDs. I hadn't adjusted it for when `createFilter()` is called from autocomplete. It's fixed now, and I added test cases too. Thanks for noticing 🙂.